config/jobs/js-ipfs/config.xml

```diff
@@ -20,11 +42,24 @@
       <triggers/>
     </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
   </properties>
-  <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.24">
-    <script>node {
-   echo &apos;Hello World&apos;
-}</script>
-    <sandbox>true</sandbox>
+  <definition class="org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition" plugin="workflow-cps@2.24">
+    <scm class="hudson.plugins.git.GitSCM" plugin="git@3.0.1">
+      <configVersion>2</configVersion>
+      <userRemoteConfigs>
+        <hudson.plugins.git.UserRemoteConfig>
+          <url>https://github.com/ipfs/js-ipfs</url>
+        </hudson.plugins.git.UserRemoteConfig>
```

Comment:
Is UserRemoteConfig the part that fetches the Jenkinsfile? It's not very obvious that it always takes the Jenkinsfile from master :)